vars/piperPipelineStageInit.groovy

```diff
@@ -46,12 +46,21 @@ void call(Map parameters = [:]) {
     def script = checkScript(this, parameters) ?: this
     def utils = parameters.juStabUtils ?: new Utils()
     def stageName = StageNameProvider.instance.getStageName(script, parameters, this)
+    def scmInfo
+    
+    node(parameters.nodeLabel?:'master') {
+        deleteDir()
 
-    piperStageWrapper (script: script, stageName: stageName, stashContent: [], ordinal: 1, telemetryDisabled: true) {
-        def scmInfo = checkout scm
+        scmInfo = checkout scm
 
         setupCommonPipelineEnvironment script: script, customDefaults: parameters.customDefaults
 
+        stash allowEmpty: true, excludes: '', includes: '**', useDefaultExcludes: false, name: 'INIT'
+        script.commonPipelineEnvironment.configuration.stageStashes = [ (stageName): [ unstash : ["INIT"]]]
```

Comment:
This creates quite an amount of file io which is not necessary in most cases.
If we cannot avoid stashing we should rather  give up on extending the init stage.
What are the use cases for extending the init stage.

Reply:
Thank you Oli for the fast review and the comment! 
You are right most users would suffer from this change. We are using the extensibility for internal use cases, but it can also be solved without an extension for the init stage. In addition we have evaluated how many users have an extension for the init stage and found out that it is not really used and for those who use it it can also be solved without an extension. Thus, I'll close this PR and we leave it as is.